packages/containers/src/DeleteResource/sagas.test.js

```diff
@@ -21,7 +21,7 @@ describe('internals', () => {
 	});
 	describe('redirect', () => {
 		it('should put a redirect action', () => {
-			const gen = internals.redirect({ data: { model: { redirectUrl: '/foo' } } });
+			const gen = internals.redirect('/foo');
```

Comment:
Those changes are purely internals or could we expect breaking changes ?

Reply:
`redirect` is used only internal, should be no breaking changes.